pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
Copying the same comment I made in kubernetes/charts: https://github.com/kubernetes/charts/pull/794#discussion_r109116948

> I like the host handling better the way the concourse chart does it, you can specify a list of hosts: https://github.com/kubernetes/charts/blob/5ba28e43b253928e374529066c65ab12ff458c56/stable/concourse/values.yaml#L288-L292
> 
> Though I agree that in most cases I'd use a single host.
>